libraries/elixir/seraphine.json

```diff
@@ -0,0 +1,20 @@
+{
+    "owner": "trilleplay",
+    "repo": "seraphine",
+    "description": "Seraphine is an API library written in Elixir for the Riot Games API.",
+    "language": "Elixir",
+    "links": [
+        {
+            "name": "Documentation",
+            "url": "https://hexdocs.pm/seraphine/readme.html"
+        },
```

Comment:
Doesnt pass Travis parsing, im guessing its due to this comma

Reply:
addressed in https://github.com/WxWatch/riot-api-libraries/pull/171/commits/a7023514b96ac51ba2879646213df1baccf3c8ab